Testing/functional/tests/cypress/integration/e2e/covid19/publicCovidTest.js

```diff
@@ -122,61 +122,66 @@ describe("Public COVID-19 Test Results", () => {
         cy.visit(covidTestUrl);
     });
 
-    it("See AB#12590", () => {
-        // Intentionally blank
+    it("Successful Response: Negative Result", () => {
+        enterFormInputs(
+            negativeCollectionDateYear,
+            negativeCollectionDateMonth,
+            negativeCollectionDateDay
+        );
+        checkResult("Negative", "Final", "text-success");
     });
```

Comment:
Looks like there's another instance of this same test below.

Reply:
I don't see the instance you refer too.

Reply:
Fixed